api/v2.0/swagger.yaml

```diff
@@ -1789,6 +1842,40 @@ paths:
           $ref: '#/responses/401'
         '500':
           $ref: '#/responses/500'
+  /projects/{project_name}/log-exts:
```

Comment:
sometimes it's `auditlog-ext` but sometimes it's `log-ext`.  Is there a reason for this inconsistency?